otter/convergence/service.py

```diff
@@ -344,7 +341,11 @@ def converge_all_groups(log, currently_converging, my_buckets, all_buckets,
     # convergence manually when it's spiraling out of control.
     # https://github.com/rackerlabs/otter/issues/1215
 
-    group_infos = yield get_my_divergent_groups(my_buckets, all_buckets)
+    if divergent_flags is None:
```

Comment:
Wait, who calls this is with None? Is that when you're just starting up? If so, why isn't that caller responsible for finding the divergent flags?